CONTRIBUTING.md

```diff
@@ -64,6 +64,21 @@ To port models from zenodo using Hugging Face hub,
 1. Create a Hugging Face account - https://huggingface.co/
 2. Request to be added to espnet organisation - https://huggingface.co/espnet
 3. Go to `egs2/RECIPE/*/scripts/utils` and run `./upload_models_to_hub.sh "ZENODO_MODEL_NAME"`
+   
+To upload models using Huggingface-cli follow the following steps:
+You can also refer to https://huggingface.co/docs/transformers/model_sharing
+1. Create a Hugging Face account - https://huggingface.co/
+2. Request to be added to espnet organisation - https://huggingface.co/espnet 
+3. Run huggingface-cli login (You can get the token request at this step under setting > Access Tokens > espnet token  
+4. huggingface-cli repo create your-model-name --organization espnet 
+5. git clone https://huggingface.co/username/your-model-name (clone this outside ESPNet to avoid issues as this a git repo)
+6. cd your-model-name
+7. git lfs install 
+8. copy contents from exp diretory of your recipe into this directory (Check other models of similar task under ESPNet to confirm your directory structure) 
```

Comment:
I think we already have the procedure at https://github.com/espnet/espnet/pull/4027/files#diff-b9f381af6d3ca8c14a68dca2a24de815830013fd69a3fe6a113e108c7aaf0d2dR1490-R1505, how about just lead to here?

Reply:
Updated the log to point to this doc

Reply:
Sorry for the misunderstanding, my issue is that we already have the `git commit` and `push` at https://github.com/espnet/espnet/pull/4027/files#diff-b9f381af6d3ca8c14a68dca2a24de815830013fd69a3fe6a113e108c7aaf0d2dR1518-R1524, so probably no need for them to proceed with your stage 9-11